source/appModules/poedit.py

```diff
@@ -79,42 +91,61 @@ def _get__dataViewControlId(self) -> int | None:
 			return None
 		return dataView.windowControlID
 
+	_sidebarControlId: int | None
+	"""Type definition for auto prop '_get__sidebarControlId'"""
+
+	def _get__sidebarControlId(self) -> int | None:
+		dataViewControlId = self._dataViewControlId
+		splitterControlID = dataViewControlId + _WindowControlIdOffsetFromDataView.MAIN_SPLITTER_IDENTIFIER
+		fg = api.getForegroundObject()
+		splitterHwnd = windowUtils.findDescendantWindow(fg.windowHandle, controlID=splitterControlID)
+		print(f"splitter: {winUser.getClassName(splitterHwnd)}")

```

Comment:
is this message useful? if so convert to a log statement else remove

```suggestion
		log.debug(f"splitter: {winUser.getClassName(splitterHwnd)}")
```

```suggestion
```